### PR TITLE
keep crashed containers

### PR DESCRIPTION
**Pull request by tbg**
[Review on Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/2974)

### CODE REVIEW

File: acceptance/localcluster/localcluster.go [hunk 1]
```diff
@@ -564,9 +564,11 @@ func (l *Cluster) stop() {
 				log.Infof("node %d: stderr at %s", i, file)
 			}
 		} else if crashed {
+			// Don't remove crashed containers - could be useful for post-mortem.
 			log.Warningf("node %d died: %s", i, ci.State)
```
Comment (tamird) [inline acceptance/localcluster/localcluster.go]: LGTM as long as this logs the container ID

File: acceptance/localcluster/localcluster.go [hunk 2]
```diff
@@ -564,9 +564,11 @@ func (l *Cluster) stop() {
 				log.Infof("node %d: stderr at %s", i, file)
 			}
 		} else if crashed {
+			// Don't remove crashed containers - could be useful for post-mortem.
```
Comment (petermattis) [inline acceptance/localcluster/localcluster.go]: I think this behavior should be under a flag and default to false or be controlled by an env variable.

Comment (tschottdorf) [inline acceptance/localcluster/localcluster.go]: if you insist, I'd rather close the PR. Adding a flag to that seems not
worth it.

Comment (petermattis) [inline acceptance/localcluster/localcluster.go]: I just worry that if you default this behavior to true then we'll have various devs ending up with tons of old docker containers lying around. I don't think controlling this via an env variable is particularly onerous.

Comment (tschottdorf) [inline acceptance/localcluster/localcluster.go]: But that doesn't matter. If you're concerned about disk space - the volumes
aren't auto-deleted, so keeping the container around isn't really a space
issue.

Comment (petermattis) [inline acceptance/localcluster/localcluster.go]: We do remove the volumes when the container is removed: https://github.com/cockroachdb/cockroach/blob/master/acceptance/localcluster/docker.go#L167

And it isn't disk space that I'm concerned about, just general cleanliness.

Comment (tschottdorf) [inline acceptance/localcluster/localcluster.go]: You're right. I still think that this isn't useful if it has to be explicitly turned on - that's not usually how you run into a node crash. I'm closing, but feel free to run with this if you feel stronger than I do that this is needed.

**Pull request closed without merging**